dockerfiles/worker.Dockerfile

```diff
@@ -1,11 +1,49 @@
-FROM python:3.6
+FROM nvidia/cuda:9.0-base-ubuntu16.04
 
-# Install PostgreSQL client
-RUN apt-get update
-RUN apt-get install -y postgresql postgresql-contrib
+RUN apt-get update && apt-get -y upgrade
 
-RUN mkdir /root/rafiki/
-WORKDIR /root/rafiki/
+# `tensorflow-gpu` dependencies
+RUN apt-get update && \
```

Comment:
is it necessary to install cuda* libs?
the nvidia docker should have already installed them.
I think you just need `pip install ...`

Reply:
In the writing of this Dockerfile, I was referring to https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/dockerfiles/dockerfiles/nvidia.Dockerfile, the official tensorflow-gpu Dockerfile. TF GPU seem to require these dependencies. Should I try removing these dependencies & test if it still works?